rhel7/profiles/hippa.xml

```diff
@@ -0,0 +1,17 @@
+<Profile id="hippa">
+<title override="true">Health Insurance Portability and Accountability Act (HIPPA) Security Rule Profile</title>
+<description override="true">
+The HIPAA Security Rule establishes national standards to protect individuals’
+electronic personal health information that is created, received, used, or
+maintained by a covered entity. The Security Rule requires appropriate
+administrative, physical and technical safeguards to ensure the
+confidentiality, integrity, and security of electronic protected health
+information. 
+
+This profile configures Red Hat Enterprise Linux 7 to the HIPPA Security
+Rule identified for securing of electronic protected health information. 
```

Comment:
single rule?

Reply:
Yes. More to come. This was OTJ training on how to create a profile.